packs/data/feats.db/perfect-resistance.json

```diff
@@ -0,0 +1,57 @@
+{
+    "_id": "QN55ToyJIjxgPmhs",
+    "img": "systems/pf2e/icons/default-icons/feats.webp",
+    "name": "Perfect Resistance",
+    "system": {
+        "actionCategory": {
+            "value": ""
+        },
+        "actionType": {
+            "value": "passive"
+        },
+        "actions": {
+            "value": null
+        },
+        "description": {
+            "value": "You gain resistance equal to half your level to all damage from effects with the appropriate trait for your School of Perfection: fire for Unblinking Flame, water for Unbreaking Waves, air for Unfolding Wind, and earth for Untwisting Iron. After using a Student of Perfection ki spell, the resistance increases to be equal to your level for 1 round.</p>\n<p>@UUID[Compendium.pf2e.feat-effects.Effect: Perfect Resistance]{Effect: Perfect Resistance}</p>"
```

Comment:
... You're gonna hate me but on re-reading this it isn't actually automatable. This isn't fire resistance, it's resistance against fire effects, water effects, etc. Not all fire damage has the fire trait (thanks Paizo) and not all things with the fire trait deal fire damage (they may do a mix of fire and bludgeoning for example). We can still set the house and see if shark wants to try and pin this down as part of IWR automation.